db/migrate/20200602095941_adds_profile_id_to_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddsProfileIdToDocuments < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :documents, :profile, type: :uuid
```

Comment:
can you add `foreign_key: true` as well

Reply:
do the models need updating with `has_many :documents` and `belongs_to :profile` etc?

Reply:
I tend to prefer to isolate migrations in a single PR and do code changes separately.

Reply:
👍